UI.AutomationTests/EmailNotifications/EmailNotificationService.cs

```diff
@@ -75,8 +75,8 @@ public async Task ValidateEmailReceived(string contactEmail, EmailTemplates emai
                 x.emailAddress == contactEmail &&
                 x.template.id == _emails[emailTemplate])
             .ToList();
-        var emailExists = notifyContactEmails.Exists(e => DateTime.Parse(e.createdAt) > DateTime.UtcNow.AddMinutes(-1));
-        Assert.That(emailExists, $"Email with template {emailTemplate} was not received by {contactEmail} in the last 60 seconds");
+        var emailExists = notifyContactEmails.Exists(e => DateTime.Parse(e.createdAt) > DateTime.UtcNow.AddMinutes(-2));
```

Comment:
2 minutes is a really really long time to verify emails being sent. this will put the test run at risk of exceeding the 60 min limit. Is this only for one test?

Reply:
It doesnt work work like that, it's filtering the query for anything within the last 3 minutes.